public/_headers

```diff
@@ -1,3 +1,6 @@
 /assets/*
-  cache-control: max-age=86400
+  cache-control: max-age=604800
+  cache-control: public
+/_astro/*
+  cache-control: max-age=604800
   cache-control: public
```

Comment:
Are we able and do we want to set headers for `/fonts/*` if we’re redirecting? We do have `public, max-age=604800, stale-while-revalidate=86400` set in the fonts-cdn so may not be needed. It would also be interesting to circle back to the CDN to see if there improvements we’d want to make.

Reply:
Yeah let me try not adding cache headers here at all, I didn't see cache headers in the `font-cdn` code base but it was late when I was playing around with this!

Reply:
Yep, that's what I missed!  I was only looking for headers in the actual edge function response

We're getting good cache hits with the redirect + the cache headers from fonts-cdn so we should be all set here :+1: